io.github.steamdeck.yml

```diff
@@ -0,0 +1,53 @@
+app-id: io.github.steamdeckutilities
```

Comment:
This needs to be a real domain controlled by the project to be listed on flathub. I just made something up for iteration purposes.

Reply:
I own cryobyte.io, which would be perfect. Does it need to have its own subdomain?

Reply:
Short answer: From my reading of the docs, the project just needs to control the root domain, so controlling io.cryobyte should be sufficient. The third part of the url will just be the project identifier, so my recommendation would be `io.cryobyte.steamdeckutilities` 

Docs: https://docs.flatpak.org/en/latest/conventions.html#application-ids